website/versioned_docs/version-latest/api_reference.mdx

```diff
@@ -137,6 +137,7 @@ There are some special variables that is available on the templating system:
 | `TIMESTAMP`        | The date object of the greatest timestamp of the files listed in `sources`. Only available within the `status` prop and if method is set to `timestamp`. |
 | `TASK_VERSION`     | The current version of task.                                                                                                                             |
 | `ITEM`             | The value of the current iteration when using the `for` property. Can be changed to a different variable name using `as:`.                               |
+| `DEFER_EXIT_CODE`  | The exit code of the last failed task. Can be used inside a defer statement to perform different operations according to the exit code.                  |
```

Comment:
Any reason why this couldn't just be `EXIT_CODE`? "Defer" is the place this variable would be used rather than a description of what the variable holds.

Reply:
No reason actually though so too, but ended up using DEFER_EXIT_CODE because it what was requested at the issue.